core/src/impl/Kokkos_ViewMapping.hpp

```diff
@@ -1030,13 +1030,13 @@ struct ViewOffset< Dimension , Kokkos::LayoutLeft
   ViewOffset( const ViewOffset< DimRHS , Kokkos::LayoutStride , void > & rhs )
     : m_dim( rhs.m_dim.N0, 0, 0, 0, 0, 0, 0, 0 )
     {
-      static_assert( 
+      /*static_assert( 
         ( DimRHS::rank == 0 &&
           dimension_type::rank == 0 ) ||
         ( DimRHS::rank == 1 &&
           dimension_type::rank == 1 &&
           dimension_type::rank_dynamic == 1 )
-        , "ViewOffset LayoutLeft and LayoutStride are only compatible when rank <= 1" );
+        , "ViewOffset LayoutLeft and LayoutStride are only compatible when rank <= 1" );*///COMMENTED: taken care by runtime check
```

Comment:
Please just delete code if you don't mean to keep it around any more.  We have version control so we can get it back if we need to :-)